packages/util/src/types.ts

```diff
@@ -127,3 +127,51 @@ export function toType<T extends TypeOutput>(
       throw EthereumJSErrorWithoutCode('unknown outputType')
   }
 }
+
+/**
+ * EIP-7702 Authorization list types
+ */
+export type EOACode7702AuthorizationListItemUnsigned = {
+  chainId: PrefixedHexString
+  address: PrefixedHexString
+  nonce: PrefixedHexString
+}
+
+export type EOACode7702AuthorizationListItem = {
+  yParity: PrefixedHexString
+  r: PrefixedHexString
+  s: PrefixedHexString
+} & EOACode7702AuthorizationListItemUnsigned
+
+// Tuple of [chain_id, address, [nonce], y_parity, r, s]
```

Comment:
Maybe you didn't mean to put the nonce in an array in the comment?
```suggestion
// Tuple of [chain_id, address, nonce, y_parity, r, s]
```